frontend/src/app/main/ui/ds/buttons/button.cljs

```diff
@@ -17,20 +17,24 @@
    [:class {:optional true} :string]
    [:icon {:optional true}
     [:and :string [:fn #(contains? icon-list %)]]]
+   [:on-ref {:optional true} fn?]
    [:variant {:optional true}
     [:maybe [:enum "primary" "secondary" "ghost" "destructive"]]]])
 
 (mf/defc button*
   {::mf/props :obj
    ::mf/schema schema:button}
-  [{:keys [variant icon children class] :rest props}]
+  [{:keys [variant icon children class on-ref] :rest props}]
```

Comment:
Why this called `on-ref`, this should be called `ref` and it should be using `forward-ref` mechanism

Reply:
This does not make sense, as it is a minor issue, i will pass it, but I'm sure that this should be solved with forward-ref